src/Resources/contao/languages/it/modules.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+$GLOBALS['TL_LANG']['MOD']['foc'][0] = "Friends of Contao";
+$GLOBALS['TL_LANG']['MOD']['focPrivacyOverview'][0]   = "Panoramica sulla privacy";
+$GLOBALS['TL_LANG']['MOD']['focPrivacyOverview'][1]   = "Qui trovate una panoramica sull\'attuale situazione della privacy nella vostra installazione";
+$GLOBALS['TL_LANG']['FMD']['analyticsOptOut'] = ['Analytics Opt-out', 'Fornisce un pulsante di disattivazione per i modelli di Analytics.'];
+$GLOBALS['TL_LANG']['FMD']['focRegistration'] = ['Rgistrazione (GDPR)', 'Genera un modulo di registrazione utente'];
```

Comment:
```suggestion
$GLOBALS['TL_LANG']['FMD']['focRegistration'] = ['Registrazione (GDPR)', 'Genera un modulo di registrazione utente'];
```

Reply:
Right.. sorry, I've a broken "e" key in my keyboard

Reply:
Ma figurati.